tests/ManageCourses.Tests/SmokeTests/DataServiceExportTests.cs

```diff
@@ -0,0 +1,148 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using GovUk.Education.ManageCourses.ApiClient;
+using GovUk.Education.ManageCourses.Tests.TestUtilities;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.UserSecrets;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+    
+namespace GovUk.Education.ManageCourses.Tests.SmokeTests
+{
+    [TestFixture]
+    public class DataServiceExportTests
+    {
+        private ApiProcessLauncher launcher = new ApiProcessLauncher();
+
+        [Test]        
+        [Category("Smoke")]
+        [Explicit]
+        public void DataExportWithEmptyCampus()
+        {
+            var config = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("integration-tests.json")
+                .AddUserSecrets<DataServiceExportTests>()
+                .AddEnvironmentVariables()
+                .Build();
+
+            var dfeSignInConfig = config.GetSection("credentials").GetSection("dfesignin");
+            
+            OrganisationCourses export = null;
+            using (launcher.LaunchApiLocally(config))
+            {                
+                var communicator = new DfeSignInCommunicator("signin-test-oidc-as.azurewebsites.net", "localhost:44364", dfeSignInConfig["clientid"], dfeSignInConfig["clientsecret"]);
+                var accessToken =  communicator.GetAccessTokenAsync(dfeSignInConfig["username"], dfeSignInConfig["password"]).Await();
+                                
+                var client = new ManageCoursesApiClient(new MockApiClientConfiguration(accessToken));
+                client.BaseUrl = "http://localhost:6001";                
+
+                client.Data_ImportAsync(TestData.MakeSimplePayload(dfeSignInConfig["username"])).Await();
+
+                Console.WriteLine($"Username: {dfeSignInConfig["username"]}");
```

Comment:
Use logger instead of console please, logger you can pipe it to data store

Reply:
oops sorry, forgot to delete